bin/glob

```diff
@@ -267,6 +267,12 @@ sub match_glob {
     return sort(@res);
 }
 
+sub getdirsep {
```

Comment:
This is an interesting way to do that, but is it better than the lookup table that File::Spec is using to decide which submodule to dispatch to?